packages/example/src/ToggleButtonExample.js

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { ToggleButton } from "@draftbit/ui";
+import Section, { Container, styles } from "./Section";
+
+export default function ToggleButtonExample() {
+  const [checked, setChecked] = React.useState("unchecked");
```

Comment:
why a string called unchecked instead of a boolean?

Reply:
That's how RN Paper does it. Check it [here](https://callstack.github.io/react-native-paper/toggle-button.html). As requested I followed how they did it. Boolean definitely makes more sense. Let me if I should change it.